lib/ansible/utils/module_docs_fragments/vmware_guest.py

```diff
@@ -0,0 +1,286 @@
+# Copyright: (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
```

Comment:
Is this moving the parts of the documentation that are shared by all `vmware_*` modules into a shared file? If so, that's awesome. Much less maintenance.

Reply:
I am moving this because it will be easier to maintain documentation in separate document fragment. This is not common to other vmware_* modules.